### DISCUSSION TITLE
Font Size

### DISCUSSION THREAD

**Discussion by opilitethesmartdev** [olikraus/u8g2#1429] [Ideas]
Could you please do  a function that sets the size of text?
Because i'm working on a OLED clock and i'm using different fonts but with different size.
And i really need that function. So please do it .

From Opilite

**Comment by olikraus** [comment]
U8g2 includes many font in different sizes. Please select a font of your desired size.

> Could you please do a function that sets the size of text?

U8g2 has been developed for small embedded systems. Adding font scaling is a requires huge resources in time and memory. 
 
> And i really need that function. 

you could port freetype and use u8g2 as underlaying render engine if you really need this

> So please do it .

I think it would take several weeks to do so. Are you willing to pay >$10000?

**Reply by olikraus** [reply]
Further reading: https://github.com/olikraus/u8g2/issues/1034
